### PR TITLE
Context refactor

### PR DESCRIPTION
**Pull request by josedahlquist**
Added new logging context functionality to the rest interface.
Added new Activity class usage in the command class to execute the handler actions.

### CODE REVIEW

File: interfaces/rest.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const Interface = require('../lib/Interface');
 // ** Platform
 const errors = require('nodus-framework').errors;
 const logger = require('nodus-framework').logging.createLogger();
```
Comment (ghost) [inline interfaces/rest.js]: should we use  require('nodus-framework').logger instead?

**Comment by ghost** [comment]
Except my comment looks good

File: interfaces/rest.js (hunk 1 above)
Comment (josedahlquist) [inline interfaces/rest.js]: good catch

**Comment by bradserbu** [comment]
👍

**Pull request merged**